src/components/keymap/keymap.component.ts

```diff
@@ -4,6 +4,8 @@ import { ActivatedRoute } from '@angular/router';
 import { SvgKeyboardPopoverComponent } from '../svg-keyboard-popover.component';
 import { Layers } from '../../../config-serializer/config-items/Layers';
 import { UhkConfigurationService } from '../../services/uhk-configuration.service';
+import { Keymap } from '../../../config-serializer/config-items/Keymap';
+import { Subscription } from 'rxjs/Rx';
```

Comment:
Change to `import { Subscription } from 'rxjs/Subscription';`
Importing from `rxjs/Rx` imports the whole Rx lib as far as I know.